tao/setup.py

```diff
@@ -34,10 +34,16 @@ def cuda_version():
     return os.environ["CUDA_VERSION"]
 
 
+def is_cpu_only():
+    if os.environ["BLADE_DISC_BUILT_CPU"]:
+        return True
+    return False
+
+
 def tensorflow_package():
+    if is_cpu_only():
+        return "tensorflow==1.15.5"
```

Comment:
what about other TF version? Maybe we could use an env variable to retrieve the version from user?

Another option is to simply rely on the TF installed in the virtual env.

Reply:
the disc wheel requires a specified version TF because of the compilation dependency of `libtao_ops.so` , so it's not easy to let users specify the TF version at the `pip install` stage or relay on the version of virtual env.